keps/sig-instrumentation/2305-metrics-cardinality-enforcement/kep.yaml

```diff
@@ -16,10 +16,12 @@ reviewers:
 approvers:
   - "@ehashman"
 creation-date: 2020-04-15
-last-updated: 2023-05-28
-stage: beta
-status: implementable
-latest-milestone: "v1.29"
+last-updated: 2024-01-20
+stage: stable
+status: implemented
```

Comment:
IIRC, we don't change this to implemented until after changes for GA have been implemented